uint8[] pickles
float32 scaler
